openmc/mgxs/mgxs.py

```diff
@@ -1998,7 +1998,7 @@ def tallies(self):
         """
 
         # Instantiate tallies if they do not exist
-        if super(NuScatterMatrixXS, self).tallies is None:
+        if super(ScatterMatrixXS, self).tallies is None:
```

Comment:
I personally think `self._tallies` is a lot more obvious (and it would have avoided this hard-to-spot bug). Just my opinion though so do what you feel is right.

Also, should `create_tallies` be private (`_create_tallies`)? Seems like a user shouldn't ever need to call that method now.